taas_api/data.py

```diff
@@ -64,3 +64,70 @@ def validate(self):
 
     def to_post_body(self):
         return {k: v for k, v in asdict(self).items() if v is not None}
+
+@dataclass
+class ChildOrder:
+    pair: str
+    side: str
+    base_asset_qty: Union[int, float, str] = None
+    quote_asset_qty: Union[int, float, str] = None
+
+    def validate(self):
```

Comment:
should we add a validation for one of base_asset_qty or quote_asset_qty and pair is a pair that exists?